app/controllers/personas_controller.rb

```diff
@@ -2,6 +2,6 @@
 
 class PersonasController < ApplicationController
   def index
-    @personas = Persona.all.decorate
+    @personas = Persona.where(service: current_service).decorate
```

Comment:
You can use the scope on the model for each service.

```suggestion
    @personas = Persona.public_send(current_service).decorate
```